src/vivarium/framework/time.py

```diff
@@ -108,12 +108,13 @@ def __init__(self) -> None:
         self._minimum_step_size: ClockStepSize | None = None
         self._standard_step_size: ClockStepSize | None = None
         self._clock_step_size: ClockStepSize | None = None
-        self._individual_clocks: PopulationView | None = None
+        self._pop_view: PopulationView | None = None
         self._pipeline_name = "simulant_step_size"
         # TODO: Delegate this functionality to "tracked" or similar when appropriate
         self._simulants_to_snooze = pd.Index([])
 
     def setup(self, builder: "Builder") -> None:
+        super().setup(builder)
```

Comment:
Same as the PopulationManager - you need to super setup to get the automatic registering of columns_created.